pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -702,11 +702,11 @@ func getReservedCPUsNum() (int, error) {
 	bootArgs := strings.Fields(string(data))
 	for _, arg := range bootArgs {
 		if strings.HasPrefix(arg, "eve_max_vcpus") {
-			argSplitted := strings.Split(arg, "=")
```

Comment:
Neither the old nor the new exists in an english dictionary, so how is this change an improvement?

Reply:
@eriknordmark isn't it split/split/split ?

Reply:
but splitArgs should be more grammatically correct